### PR TITLE
final evaluation with peak+LUFS normalization for both CHiME and Libr…

### PR DESCRIPTION
**Pull request by msaadeghii**
The new script final_evaluation.py evaluates on both chime and librichime with peak + lufs normalization prior to dns-mos.

### CODE REVIEW

File: baseline/utils/final_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+                student_estimates = model(input_mix)
+                student_estimates = mixture_consistency.apply(student_estimates, input_mix)
+
+                s_est_speech = student_estimates[0, 0, :file_length].detach().cpu().numpy().squeeze()
+
+            # Peak normalization to 0.7
+            s_est_speech -= s_est_speech.mean()
+            s_est_speech /= np.abs(s_est_speech).max() + 1e-9
+            s_est_speech *= 0.7
```
Comment (etzinis) [inline baseline/utils/final_evaluation.py:154]: Why this value?

Comment (etzinis) [reply baseline/utils/final_evaluation.py:154]: At least comment here, and explain why this value, also make sure the same normalization is applied during training.

**Review by etzinis — CHANGES_REQUESTED** [review]

File: baseline/utils/final_evaluation.py (hunk 1 above)
Comment (msaadeghii) [reply baseline/utils/final_evaluation.py:154]: This is just to make sure that clipping does not happen after loudness normalization. I added a comment, and modified the training script as well.

Comment (etzinis) [reply baseline/utils/final_evaluation.py:154]: Please write the reason that you chose to use 0.7 for peak normalization.
